src/google/protobuf/compiler/csharp/csharp_message.cc

```diff
@@ -123,6 +130,10 @@ void MessageGenerator::Generate(io::Printer* printer) {
   printer->Print(
       "private pb::UnknownFieldSet _unknownFields;\n");
 
+  for (int i = 0; i < has_bit_field_count_; i++) {
```

Comment:
Thanks for explain. Please add comments here to explain why use ints here instead of bitsarray or bits.